.github/workflows/plugin-packages.yml

```diff
@@ -71,14 +70,12 @@ jobs:
               - '.krew/tvk-log-collector.yaml'
               - 'tools/log-collector/**'
               - '.goreleaser.yml'
-              - 'tests/common/**'
             testPreflight:
               - 'tools/preflight/**'
               - 'tests/preflight/**'
             testLogCollector:
```

Comment:
remove this whole `testLogCollector` section and wherever it's used

Reply:
resolved